content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -45,71 +156,46 @@ team. These credentials are only used to generate the
+Finally, use the `dvc remote modify` command to set the credentials (for each
+GDrive remote), for example:
 
 ```dvc
-$ dvc remote add -d mygdroot gdrive://root
-$ dvc remote add mygdfolder gdrive://root/path/to/folder
+$ dvc remote add gdfolder gdrive://0AIac4JZqHhKmUk9PDA
+$ dvc remote modify gdfolder gdrive_client_id <client ID>
+$ dvc remote modify gdfolder gdrive_client_secret <client secret>
 ```
 
-2. Google Drive _Folder ID_
-
-To obtain the unique _Folder ID_ in question, navigate into that folder in your
-web browser, and find it in the address bar. For example, for
-`https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`:
-
-```dvc
-$ dvc remote add mygdfolder gdrive://0AIac4JZqHhKmUk9PDA
-$ dvc remote add mygdsubfolder \
-                 gdrive://0AIac4JZqHhKmUk9PDA/sub/path
-```
+> Note that Google Drive API usage limits/quotas apply per _project_ client and
+> can be reviewed in the
+> [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent).
+> Please keep this in mind when sharing them, or you may
+> [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
 
-[Shared drives](https://developers.google.com/drive/api/v2/about-shareddrives)
-should be referenced by _Folder ID_ in the same way.
+## Using service accounts
 
-3. `appDataFolder` - special
-   [application-specific data](https://developers.google.com/drive/api/v2/appdata)
-   folder only accessible by your application and hidden from the user.
+A [service account](https://cloud.google.com/iam/docs/service-accounts) is a
+Google account associated with your GCP project, and not a specific user. They
+are intended for scenarios where your code needs to access data on its own, e.g.
+running inside a Compute Engine, automatic CI/CD, etc. No interactive user OAuth
+authentication is needed.
 
-```dvc
-$ dvc remote add mygdappata gdrive://appDataFolder
-```
+> This requires having your own
+> [GC project](/doc/user-guide/setup-google-drive-remote#connect-using-a-custom-google-cloud-project)
+> as explained above.
 
-## Configure the GDrive remote
+1. To
+   [create a service account](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account),
+   navigate to **IAM & Admin** in the left sidebar, and select **Service
+   Accounts**. Click **+ CREATE SERVICE ACCOUNT**, on the next screen, enter
+   **Service account name** e.g. "My DVC project", and click **Create**. Select
+   **Continue** at the next **Service account permissions** page, click at **+
+   CREATE KEY**, select **P12** and **Create**. Save generated `.p12` key file
+   at your local disk.
 
-Use the `dvc remote modify` command to set the credentials for each `gdrive://`
-remote, for example:
+1. Copy a downloaded `.p12` file to your DVC project root directory.
 
 ```dvc
-$ dvc remote add -d mygdfolder gdrive://root/path/to/folder
```

Comment:
it looks like we are missing `3. Setup DVC to use ...`

Reply:
I'm not sure on the location of this note since it's left in a removed line and the line count has changed a lot. But if this was about the Service Acct section, I think it's no longer needed.